content/3_Silver/DFS.mdx

```diff
@@ -585,3 +585,77 @@ public class BuildingTeams
 ### Problems
 
 <Problems problems="bip" hideSuggestProblemButton />
+
+## Quiz
+<Quiz>
+	<Quiz.Question>
+		What would be a valid function header for DFS on an undirected tree to work in linear time *without* any other variables?
+		<Quiz.Answer correct>
+			`void dfs(int v, int p, vector<vector<int> >& adj)`
```

Comment:
```suggestion
			`void dfs(int v, int p, vector<vector<int>>& adj)`
```

I don't think these spaces are necessary post C++98?

Reply:
honestly it's just a preference thing